src/main/java/core/basesyntax/ReadableFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public interface ReadableFile {
```

Comment:
Why do we need this interface. I can't see any variable declaration for its type.

Reply:
done